### PR TITLE
chore(repo): use new launch template syntax

### PR DESCRIPTION
**Pull request by brian-nrwl**

### CODE REVIEW

**Review by FrozenPandaz — APPROVED** [review]

File: .nx/workflows/agents.yaml [hunk 1]
```diff
@@ -1,59 +1,60 @@
...
+      NX_DAEMON: 'true'
+      NX_PERF_LOGGING: 'false'
+      NX_NATIVE_LOGGING: 'false'
+      SELECTED_PM: 'pnpm'
+      NX_E2E_RUN_E2E: 'true'
+      NPM_CONFIG_PREFIX: '/home/workflows/.npm-global'
+      NX_CLOUD_ACCESS_TOKEN: '{{secrets.NX_CLOUD_ACCESS_TOKEN}}'
+    on:
+      - events:
+          - CREATE_RUN_GROUP
```
Comment (rarmatei) [inline .nx/workflows/agents.yaml:20]: we shouldn't need this anymore

File: .nx/workflows/agents.yaml [hunk 2]
```diff
@@ -1,59 +1,60 @@
...
+      NX_VERBOSE_LOGGING: 'false'
+      NX_DAEMON: 'true'
+      NX_PERF_LOGGING: 'false'
+      NX_NATIVE_LOGGING: 'false'
+      SELECTED_PM: 'pnpm'
+      NX_E2E_RUN_E2E: 'true'
+      NPM_CONFIG_PREFIX: '/home/workflows/.npm-global'
+      NX_CLOUD_ACCESS_TOKEN: '{{secrets.NX_CLOUD_ACCESS_TOKEN}}'
+    on:
+      - events:
```
Comment (rarmatei) [inline .nx/workflows/agents.yaml:19]: I think we can delete the events section

File: .nx/workflows/agents.yaml [hunk 3]
```diff
@@ -1,59 +1,60 @@
...
+          - CREATE_RUN_GROUP
+    steps:
```
Comment (rarmatei) [inline .nx/workflows/agents.yaml:21]:
```suggestion
    init-steps:
```

File: .nx/workflows/agents.yaml [hunk 4]
```diff
@@ -1,59 +1,60 @@
...
+          git config --global user.name "Test Test"
 
-  - name: Run Agent
-    script: |
-      source "$HOME/.cargo/env"
-      npx nx-cloud start-agent
+      - name: Run Agent
+        script: |
+          source "$HOME/.cargo/env"
+          npx nx-cloud start-agent
```
Comment (rarmatei) [inline .nx/workflows/agents.yaml:60]: we don't need start-agent anymore

**Pull request merged**